src/main/kotlin/ar/edu/unahur/obj2/socios/Cliente.kt

```diff
@@ -1,4 +1,45 @@
 package ar.edu.unahur.obj2.socios
 
-class Cliente {
+
+object Cliente { //Patron Singleton aplicado
+    lateinit var estadoDeAnimo: EstadoDeAnimo //Los atributos del creador fueron cambiados por 'lateinit var' ya que se van a declarar cuando sea necesario
+    var bolsillo: Double = 0.0
+    lateinit var pedido : Pedido
+    lateinit var barrio: Barrio
+
+    //Getters
+    fun consultarPropina() = this.estadoDeAnimo.propina(this)
+    fun consultarPropinaBarrio() = this.barrio.cuenta(this.consultarPropina())
```

Comment:
🤔 Por qué lo separaron?? 
`consultarPropina()` tendría que hacer lo que hace `consultarPropinaBarrio()`, el cálculo de la propina depende tanto del animo cómo el barrio en el que vive.
```suggestion
    fun consultarPropina() = this.barrio.cuenta(this.consultarPropina())
```